ci/run_python_tests.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -euo pipefail
+
+TEST_CMD=$1
+TEST_ARGS=$2
```

Comment:
If you want to follow the best practice. I think those vars are local and can be lower case, but just a suggestion not a hard stop :)
TEST_CMD="${1:-}"
TEST_ARGS="${2:-}"

And then check for amount of positional arguments are you expectin
[ "$#" -ne 2 ] && { echo -e "Error: function requires exactly two arguments.\n\n"; exit 1;}